spec/api_callers/http_request_spec.rb

```diff
@@ -2,22 +2,36 @@
 require 'net/http'
 
 describe ::ApiCallers::HttpRequest do
-  subject { ::ApiCallers::HttpRequest.new('') }
+  subject { ::ApiCallers::HttpRequest.new('some_uri', http_verb) }
+  let(:http_verb) { 'get' }
 
-  describe "#headers" do
-    it "returns { 'Accept-Charset' => 'utf-8' }" do
-      expect(subject.send(:headers)).to eq({ 'Accept-Charset' => 'UTF-8' })
+  describe "#make_request" do
+    let(:http_obj) { double('http', request: request) }
+    let(:request) { double('request', body: { response: 'response'}) }
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.